lib/palindrome_check.rb

```diff
@@ -0,0 +1,6 @@
+# A method to check if the input string is a palindrome.
+# Return true if the string is a palindrome. Return false otherwise.
+def palindrome_check(my_phrase)
+  puts "NOT IMPLEMENTED"
+  return true
```

Comment:
How would you feel about doing `raise NotImplementedError` here instead?

Best practice I think is generally to avoid `puts` statements and other things that will print to the console for code being tested, because that output kind of gets randomly interspersed with the test runner's output.